lib/manager/composer/artifacts.spec.ts

```diff
@@ -85,6 +94,80 @@ describe('.updateArtifacts()', () => {
       })
     ).toBeNull();
     expect(execSnapshots).toMatchSnapshot();
+    expect(fs.outputFile.mock.calls[1][1]).toMatchSnapshot();
```

Comment:
This test even though it's name is `uses hostRules to write auth.json` never asserted what is actually written to `auth.json`. So I added one more expectation.